docs/Installation.md

```diff
@@ -291,6 +294,20 @@ Initialize the database:
 mysql --user=root --password < ./initial-mysql.sql
 ```
 
+For latest versions of MySQL, wherein the root password is blank and you get an
+error "Access denied for the above command" on running the above command ,
+follow the below procedure: 

```

Comment:
@sandoche2k, have you tried my suggestion? I think the answer is simpler than that. If the password is blank I think you can connect to the database by omitting the password switch:

```sh
 mysql --user=root < ./initial-mysql.sql
 ```